tests/Provider/BuilderAliasProviderTest.php

```diff
@@ -218,11 +219,21 @@ private function createMockKernelForStub()
         ;
 
         $kernel = $this->getMockBuilder('Symfony\Component\HttpKernel\KernelInterface')->getMock();
-        $kernel->expects($this->once())
-            ->method('getBundle')
-            ->with('FooBundle', false)
-            ->will($this->returnValue(array($bundle)))
-        ;
+        // Fix deprecation notice on Symfony 3.4 and higher, assuring BC
+        $version = Kernel::MAJOR_VERSION.Kernel::MINOR_VERSION;
+        if ($version >= BuilderAliasProvider::VERSION_DEPRECATION) {
+            $kernel->expects($this->once())
+                ->method('getBundle')
+                ->with('FooBundle')
+                ->will($this->returnValue(array($bundle)))
```

Comment:
This will return not the array but the `$bundle` as object. However that's not what causing the error. I'm trying to figure out what's the cause.